### PR TITLE
Graduation Year Book Application

### PR DESCRIPTION
**Pull request by tiwaripramod99**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] ~~**Tier 2** ✏️: I wrote a post on DEV about a project I built.~~
  - [ ] ~~**Tier 3** ✨: I want my project to be highlighted on stream.~~
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/tiwaripramod99/tiwaripramod99.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Pramod Tiwari # No longer than 18 characters
+institution: U.V Patel College of Engineering # no longer than 58 characters
+profile_pic: tiwaripramod99.png # Name and extension of your profile picture(ex. mona.png)
+quote: People who are crazy enough to think that they can change the world are the one's who do. Programmers being one of those. # no longer than 100 characters
```
Comment (CLiu13) [inline _data/tiwaripramod99/tiwaripramod99.md:5]: Your quote does not meet the character length requirement (100 characters max, current character count: 121).

File: _data/tiwaripramod99/tiwaripramod99.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: Pramod Tiwari # No longer than 18 characters
+institution: U.V Patel College of Engineering # no longer than 58 characters
+profile_pic: tiwaripramod99.png # Name and extension of your profile picture(ex. mona.png)
```
Comment (CLiu13) [inline _data/tiwaripramod99/tiwaripramod99.md:4]: The file extension does match (image is a ```.jpg```, not a ```.png```).

File: _data/tiwaripramod99/tiwaripramod99.md (hunk 1 above)
Comment (tiwaripramod99) [reply _data/tiwaripramod99/tiwaripramod99.md:5]: Cannot make changes to this PR since it's already closed

Comment (juanpflores) [reply _data/tiwaripramod99/tiwaripramod99.md:5]: You can open a new PR

Comment (tiwaripramod99) [reply _data/tiwaripramod99/tiwaripramod99.md:5]: I did, but the form has been closed. And I believe that's the reason it hasn't been reviewed yet.
Here's a link to the new PR.
https://github.com/education/graduation/pull/4179

**Pull request closed without merging**